### PR TITLE
Add features

### PR DESCRIPTION
**Pull request by ViachaslauKabak**
* Remove DB driver download dirty hack
* Add migration module usage
* Add json usage for init and DB update

### CODE REVIEW

File: files/opt/hybris-tools/h-up [hunk 1]
```diff
@@ -273,8 +268,22 @@ if __name__ == "__main__":
                 fout.writelines(data)
             
         if config["h_init"]:
-            ant_log_out.append(h_ant("initialize", config["username"], config["hybris_custom_ant_targets_path"]))
+            value_type = type(config["h_init"])
+            if value_type is bool:
```
Comment (kharkevich) [inline files/opt/hybris-tools/h-up]: isinstance(value_type, bool) should be used

File: files/opt/hybris-tools/h-up [hunk 2]
```diff
@@ -273,8 +268,22 @@ if __name__ == "__main__":
...
+                ant_log_out.append(h_ant("initialize", config["username"], config["platform_home"]))
+            elif value_type is str:
+                ant_log_out.append(h_ant("initialize -DconfigFile=" + config["h_init"], config["username"],
```
Comment (kharkevich) [inline files/opt/hybris-tools/h-up:274]: isinstance(value_type, str) should be used
ref: https://www.peterbe.com/plog/bool-is-int

File: files/opt/hybris-tools/h-up [hunk 3]
```diff
@@ -273,8 +268,22 @@ if __name__ == "__main__":
...
+                                         config["platform_home"]))
         if config["h_sysupdate"]:
+            # h_sysupdate value can be bool (True, without config file) or string (path to custom config file)
+            # In case of string value (path to custom config file) we will call ant updatesystem with -DconfigFile param
+            value_type = type(config["h_sysupdate"])
+            if value_type is bool:
```
Comment (kharkevich) [inline files/opt/hybris-tools/h-up]: isinstance(value_type, bool) should be used

File: files/opt/hybris-tools/h-up [hunk 4]
```diff
@@ -273,8 +268,22 @@ if __name__ == "__main__":
...
+                ant_log_out.append(h_ant("updatesystem", config["username"], config["platform_home"]))
+            elif value_type is str:
```
Comment (kharkevich) [inline files/opt/hybris-tools/h-up]: isinstance(value_type, str) should be used

**Review by kharkevich — CHANGES_REQUESTED** [review]

File: files/opt/hybris-tools/h-up (hunk 1 above)
Comment (ViachaslauKabak) [reply files/opt/hybris-tools/h-up]: fixed

**Review by kharkevich — APPROVED** [review]

**Pull request merged**